_articles/keyboard-shortcuts.md

```diff
@@ -42,6 +42,8 @@ Enter the information about your shortcut, including what the name will be, the
 
 Then click "Add", and your shortcut will be active.
 
+*Note*: For executing shell script, use absolute path instead of relative path. Example: `sh /home/username/Documents/script.sh` instead of `sh ~/Documents/script.sh`
```

Comment:
Thanks for the contribution! This sentence reads a little strangely to me. Do you think "In order to execute a shell script, use its absolute path instead of its relative path" would sound better?

Reply:
Ah yes, it will be better that way 👍.